requirements/dev-requirements.txt

```diff
@@ -8,6 +8,7 @@ git+git://github.com/dimagi/couchdbkit-debugpanel.git
 Fabric==1.10.0
 ansible==1.8.2  # to run fab development *
 Sphinx==1.2b1
+jenkinsapi=0.2.28
```

Comment:
I'm trying this out locally, so I just pushed a fix for that and the merge conflict.